pyiron_base/job/script.py

```diff
@@ -31,6 +31,39 @@ class ScriptJob(GenericJob):
         project (ProjectHDFio): ProjectHDFio instance which points to the HDF5 file the job is stored in
         job_name (str): name of the job, which has to be unique within the project
 
+    Example:
+        To submit a notebook as a ScriptJob job to the pyiron job management system,
+
+            1. Create the notebook to be submitted, for ex. 'example.ipynb', and save it -- Can contain any code like:
+                ```
+                from pyiron import Project
+                pr = Project('.')
```

Comment:
Since we don't really recommend to use `Project('.')` in general, it's probably better either to give a name or use `default_working_directory=True`.

Reply:
I agree, it does look a little odd. I will change this.